### PR TITLE
Don't use DOTNET_BUILD_VERSION in code.

### PR DESCRIPTION
**Pull request by pranavkm**
Clean up test projects

### CODE REVIEW

File: test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs [hunk 1]
```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (victorhurdugaci) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: Why did you add this class?

Comment (pranavkm) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: We needed to change the `version` portion of  `AddToolToProject`. It didn't look like this type was used in any other project so it makes sense to localize it.

Comment (victorhurdugaci) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: Just change the other code and make the version a parameter. The code is in the testing repo.

Comment (pranavkm) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: Why is it in a separate repo?

Comment (victorhurdugaci) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: Because it should be used for the iis tool tests.

Comment (pranavkm) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: IIS tools don't have functional tests as yet. Do you plan on using it there? Additionally, the semantics there is pretty different - you're primarily publishing the application and watching the side effects as opposed to running the tool as standalone. It's pretty unlikely a lot of this is re-usable in that context.

Comment (pranavkm) [inline test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs]: Bump.

**Comment by victorhurdugaci** [comment]
:shipit:
